keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -142,10 +212,15 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
     be either an integer or string type.
 
     Args:
-        vocabulary: A list of strings or a string filename path. If
-            passing a list, each element of the list should be a single
-            WordPiece token string. If passing a filename, the file should be a
-            plain text file containing a single WordPiece token per line.
+        vocabulary: A list of strings or a string filename path. If passing a
+            list, each element of the list should be a single WordPiece token
+            string. If passing a filename, the file should be a plain text file
+            containing a single WordPiece token per line.
+        lang: An ISO 639-1 language code. Loads the tokenizer with a vocabulary
```

Comment:
Add a link

Reply:
I think we should specifically tie ourselves to wikipedias list of languages (those are our datasets). Is iso 693-1 the same thing?

Reply:
Docstring should clearly state that specifying `lang` loads a pretrained vocabulary. It would also be nice to understand what data was used as this is pretty mysterious as presented.